### ISSUE TITLE
Error opening dats.

### ISSUE THREAD

**Issue by BearFather** [syntax53/Nightmare-Redux#8]
So a few days ago, nightmare has been unable to load my backups from my server.  I have a backup from the 7th works great, then load the 10th, I get btrieve error about can't find the file.  File exists and is not zero length. Same folders, and same file permissions. 

This happens on my windows 10 machine.  But my windows 2000 server it open's fine.

**Comment by syntax53** [comment]
Did you update your dats to the new btrieve version?  This sounds like a btrieve dll issue.  Either an old dll trying to access new dats, or a new dll that isn't properly setup on the computer.

**Comment by BearFather** [comment]
Nope standard btrieve, I never did update them.

**Comment by syntax53** [comment]
both boxes running v1.8 accessing the same backup dats?
neither box the one the board is running on?
-- if so: delete all traces of WBTRV32.DLL and W32MK*.* (3 files) from nightmare folder and system folders.  Probably found under 3 possible locations... c:\windows\system32, C:\windows\SysWOW64, C:\Program Files (x86)\Nightmare Redux (or non x86 folder on 32bit pc).  And then reinstall NMR.

**Comment by BearFather** [comment]
Yes both ran are running 1.8.  Both were updated from 1.7 to 1.8.  I have uploaded the file from my live server to dropbox and downloaded them both and extracted.  On my windows 10 machine the older and newer backup are in the same folder, and even tried to run as admin but no dice.

Went thru and deleted all the files in SysWOW64 and reinstalled and I was able to load the 10th backup.  Then checked the 12th backup and it didn't work.  So I got one more out of it. 

I just downloaded last night backup and it will open it now with no issue.  But my 12th backup it won't open but it will open the 10th and 13th.
